Decisions.Equifax/CodeAndDescriptionDataConverter.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Linq;
+using Decisions.Equifax.PreQualificationOfOne.Response;
+using Newtonsoft.Json;
+using JsonConverter = Newtonsoft.Json.JsonConverter;
+
+namespace Decisions.Equifax
+{
+    internal class CodeAndDescriptionDataConverter : JsonConverter
+    {
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            serializer.Serialize(writer, value);
+        }
+
+        public override CodeAndDescription ReadJson(JsonReader reader, Type objectType, object existingValue,
+            JsonSerializer serializer)
+        {
+            if (reader.TokenType == JsonToken.StartArray)
+            {
+                CodeAndDescription deserializeResult =
+                    serializer.Deserialize<CodeAndDescription[]>(reader).FirstOrDefault();
```

Comment:
This could potentially return null. Are you handling the possibility of returning null or will an error be thrown?